rbac/management/cache.py

```diff
@@ -34,6 +34,96 @@ def connection(self):
                 raise
         return self._connection
 
+    @contextlib.contextmanager
+    def delete_handler(self, err_msg):
+        """Handle delete events."""
+        try:
+            yield
+        except exceptions.RedisError:
+            logger.exception(err_msg)
+
+    def get_from_redis(self, index):
+        """Get object from redis based on index."""
+        obj = self.connection.hget(*(self.key_for(index[0]), index[1]))
```

Comment:
This wasn't terribly clear to me, either, what the `index[0]` / `index[1]` values are, and what the expected interface is for them. Again, maybe a case of asking whether or not we should be assuming `h*` are the methods to use by default? Just throwing it out there for consideration. 😄